airflow/www/static/css/main.css

```diff
@@ -238,6 +238,19 @@ div.square {
   box-shadow: inset 0 6px 6px rgba(0, 0, 0, 0.4);
 }
 
+#timezone-menu li > span {
```

Comment:
I don't see a `span` that is a direct descendant of an `li` in the markup?

Ideal to avoid generic selectors all together, use a class instead. e.g. `.timezone-menu-label`.

Reply:
`<li id="timezone-other"><span>Other</span>`, but yes point taken, I'll update this to a class instead.

Reply:
ah, completely missed it being on the same line as the `<li>` there!